ui/pages/permissions-connect/snaps/snap-result/snap-result.js

```diff
@@ -91,7 +86,7 @@ export default function SnapResult({
         </Text>
         <Text textAlign={TextAlign.Center}>
           {t('snapResultSuccessDescription', [
-            <Text as={ValidTag.Span} key="1" fontWeight={FontWeight.Medium}>
+            <Text as="span" key="1" fontWeight={FontWeight.Medium}>
```

Comment:
Non MR related question: do we need this key here? and assigning it the same value is not an issue?